pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -30,7 +30,7 @@ func checkVersionOfWindowsUpdate() (bool, error) {
 	}
 
 	if yourWindowsReleaseId < minimumWindowsReleaseId {
-		return false, errors.Newf("Please update Windows. Currently %d is the minimum release needed to run. You are running %d", minimumWindowsReleaseId, yourWindowsReleaseId)
+		return false, errors.New(fmt.Sprintf("Please update Windows. Currently %d is the minimum release needed to run. You are running %d", minimumWindowsReleaseId, yourWindowsReleaseId))
```

Comment:
@gbraad Since preflight checks are directly run from the `start.go` so the errors are wrapped with `errors.Newf` which is the part of `github.com/code-ready/crc/pkg/crc/errors` package but you now using standard `errors` package which not going to print out in stdout when the issue occur.

Reply:
I do not prefer to see:

```
FAIL: Error occured: Please update Windows. Currently 1703 is the minimum release needed to run. You are running ..
FAIL Please update Windows. Currently 1703 is the minimum release needed to run. You are running .."
```

which duplicates messages.

Reply:
The output with `crc/errors` is as follows:
![image](https://user-images.githubusercontent.com/1894/64613581-109c8c80-d409-11e9-9e79-c5be65d971b4.png)

Perhaps the last message is NEVER to be printed?

Reply:
Regarding use of errors.Newf, my understanding was that only Start() would call it, that methods called from Start() should return 'regular' errors instead.

Reply:
With using `errors` it is:
![image](https://user-images.githubusercontent.com/1894/64613769-8e609800-d409-11e9-9cd8-f51cca8a4212.png)

Reply:
Regression perhaps?